deis-cluster-coreos/azuredeploy.json

```diff
@@ -0,0 +1,361 @@
+{
+    "$schema": "http://schema.management.azure.com/schemas/2014-04-01-preview/deploymentTemplate.json",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "newStorageAccountName": {
+            "type": "string",
+            "metadata": {
+                "description": "Name of the of the storage account for VM OS Disk"
+            }
+        },
+        "vmNamePrefix": {
+            "type": "string",
+            "defaultValue": "coreos",
+            "metadata": {
+                "description": "Prefix for the names of the VMs"
+            }
+        },
+        "vmSize": {
+            "type": "string",
+            "defaultValue": "Standard_A1",
+            "metadata": {
+                "description": "Instance size for the VMs"
+            }
+        },
+        "adminUserName": {
+            "type": "string",
+            "metadata": {
+                "description": "Username to login to the VMs"
+            }
+        },
+        "sshKeyData": {
+            "type": "string",
+            "metadata": {
+                "description": "Public key for SSH authentication"
+            }
+        },
+        "customData": {
+            "type": "string",
+            "metadata": {
+                "description": "Base64-encoded cloud-config.yaml file to deploy and start Fleet"
+            }
+        },
+        "numberOfNodes": {

```

Comment:
At this point it's limited to 3 because of the inbound NAT rule syntax. I'm following this up on Yammer and will fix if a solution is available.